test/msw-api/setup-worker/setupWorker.test.ts

```diff
@@ -0,0 +1,25 @@
+import * as path from 'path'
```

Comment:
Lets rename this test suite so it’s clear what it does: `input-validation.test.ts`.